components/properties/src/runtime.rs

```diff
@@ -0,0 +1,436 @@
+            "Changes_When_Casefolded" | "CWCF" => Self::ChangesWhenCasefolded,
+            "Changes_When_Casemapped" | "CWCM" => Self::ChangesWhenCasemapped,
+            "Changes_When_Lowercased" | "CWL" => Self::ChangesWhenLowercased,
+            "Changes_When_NFKC_Casefolded" | "CWKCF" => Self::ChangesWhenNfkcCasefolded,
+            "Changes_When_Titlecased" | "CWT" => Self::ChangesWhenTitlecased,
+            "Changes_When_Uppercased" | "CWU" => Self::ChangesWhenUppercased,
+            "Dash" => Self::Dash,
+            "Default_Ignorable_Code_Point" | "DI" => Self::DefaultIgnorableCodePoint,
+            "Deprecated" | "Dep" => Self::Deprecated,
+            "Diacritic" | "Dia" => Self::Diacritic,
+            "Emoji" => Self::Emoji,
+            "Emoji_Component" | "EComp" => Self::EmojiComponent,
+            "Emoji_Modifier" | "EMod" => Self::EmojiModifier,
+            "Emoji_Modifier_Base" | "EBase" => Self::EmojiModifierBase,
+            "Emoji_Presentation" | "EPres" => Self::EmojiPresentation,
+            "Extended_Pictographic" | "ExtPict" => Self::ExtendedPictographic,
+            "Extender" | "Ext" => Self::Extender,
+            "Grapheme_Base" | "Gr_Base" => Self::GraphemeBase,
+            "Grapheme_Extend" | "Gr_Ext" => Self::GraphemeExtend,
+            "Hex_Digit" | "Hex" => Self::HexDigit,
+            "IDS_Binary_Operator" | "IDSB" => Self::IdsBinaryOperator,
+            "IDS_Trinary_Operator" | "IDST" => Self::IdsTrinaryOperator,
+            "ID_Continue" | "IDC" => Self::IdContinue,
+            "ID_Start" | "IDS" => Self::IdStart,
+            "Ideographic" | "Ideo" => Self::Ideographic,
+            "Join_Control" | "Join_C" => Self::JoinControl,
+            "Logical_Order_Exception" | "LOE" => Self::LogicalOrderException,
+            "Lowercase" | "Lower" => Self::Lowercase,
+            "Math" => Self::Math,
+            "Noncharacter_Code_Point" | "NChar" => Self::NoncharacterCodePoint,
+            "Pattern_Syntax" | "Pat_Syn" => Self::PatternSyntax,
+            "Pattern_White_Space" | "Pat_WS" => Self::PatternWhiteSpace,
+            "Quotation_Mark" | "QMark" => Self::QuotationMark,
+            "Radical" => Self::Radical,
+            "Regional_Indicator" | "RI" => Self::RegionalIndicator,
+            "Sentence_Terminal" | "STerm" => Self::STerm,
+            "Soft_Dotted" | "SD" => Self::SoftDotted,
+            "Terminal_Punctuation" | "Term" => Self::TerminalPunctuation,
+            "Unified_Ideograph" | "UIdeo" => Self::UnifiedIdeograph,
+            "Uppercase" | "Upper" => Self::Uppercase,
+            "Variation_Selector" | "VS" => Self::VariationSelector,
+            "White_Space" | "space" => Self::WhiteSpace,
+            "XID_Continue" | "XIDC" => Self::XidContinue,
+            "XID_Start" | "XIDS" => Self::XidStart,
+            _ => return None,
+        };
+
+        Some(prop)
+    }
+
+    ///
+    pub fn load_ecma402_binary_property_unstable<P>(
```

Comment:
unsure of these names

Reply:
Since there are only _two_ "normal" enumerated properties, and from an FFI standpoint they have different types (u8 vs u16), I've decided to not add the corresponding enumerated property API and let people do their own thing instead.